RadarSDK/RadarLocationManager.m

```diff
@@ -165,17 +166,18 @@ - (void)startTrackingWithOptions:(RadarTrackingOptions *)trackingOptions {
     CLAuthorizationStatus authorizationStatus = [self.permissionsHelper locationAuthorizationStatus];
     if (!(authorizationStatus == kCLAuthorizationStatusAuthorizedWhenInUse || authorizationStatus == kCLAuthorizationStatusAuthorizedAlways)) {
         [[RadarDelegateHolder sharedInstance] didFailWithStatus:RadarStatusErrorPermissions];
-
         return;
     }
 
     [RadarSettings setTracking:YES];
     [RadarSettings setTrackingOptions:trackingOptions];
+    //[RadarSettings setFallbackTrackingOptions:trackingOptions];
```

Comment:
hmm why is this commented out? this is how we'd "fallback" to the client-side configued tracking options

Reply:
aha, I see the answer below in

```
[Radar getTrackingOptions]
```

okay, that works for me. let's just remove this, then.

Reply:
Removed.